.kokoro/tests/run_tests.sh

```diff
@@ -55,13 +55,20 @@ fi
 # quota related errors. I think we can avoid this by changing the
 # order of tests to execute (e.g. reverse order for py-3.8
 # build). Currently there's no easy way to do that with btlr, so we
-# temporarily wait few minutes to avoid quota issue for py-3.8
-# presubmit build.
-if [[ "${RUN_TESTS_SESSION}" == "py-3.8" ]] \
-       && [[ "${KOKORO_JOB_NAME}" == *presubmit ]]; then
-    echo -n "Detected py-3.8 presubmit build,"
-    echo "Wait few minutes to avoid quota issues."
-    sleep 5m
+# temporarily wait few minutes to avoid quota issue for some
+# presubmit builds.
+if [[ "${KOKORO_JOB_NAME}" == *presubmit ]] \
+       && [[ -z "${DIFF_FROM:-}" ]]; then
+    if [[ "${RUN_TESTS_SESSION}" == "py-3.7" ]]; then
+	echo -n "Detected py-3.7 presubmit full build,"
+	echo "Wait 5 minutes to avoid quota issues."
+	sleep 5m
```

Comment:
Should these sleeps perhaps be configurable via an env var rather than session specifc?

Reply:
Well, technically it's possible, but I think it's a temporary workaround until btlr supports list of directories. Once btlr supports a list of directories, this effort will be moot.